packages/libs/components/src/plots/VolcanoPlot.tsx

```diff
@@ -155,6 +161,8 @@ function VolcanoPlot(props: VolcanoPlotProps, ref: Ref<HTMLDivElement>) {
     []
   );
 
+  const effectSizeLabel = data?.effectSizeLabel;
```

Comment:
I think we can remove the question mark here since we're passing in a default value to the `data` prop that defines the `effectSizeLabel` property.